Common/TVSeries.ArtworkChooser.xml

```diff
@@ -152,7 +152,7 @@
    		<posY>150</posY>
    		<width>250</width>
    		<height>140</height>
-   		<texture>..\..\..\Thumbs\ClearArt\Series\#(string.trim(#TVSeries.Series.ID)).png</texture>
+   		<texture>#fanarthandler.series.clearart.selected</texture>

```

Comment:
Need to change from `#fanarthandler.series......` to `#fanarthandler.tvseries.....`

Reply:
Completed